core/constraints/constraints_test.go

```diff
@@ -337,6 +337,9 @@ var parseConstraintsTests = []struct {
 	}, {
 		summary: "multiple zones",
 		args:    []string{"zones=az1,az2"},
+	}, {
+		summary: "Spaced zones",
+		args:    []string{"zones=Availability zone 1"},
```

Comment:
@Pekkari great work, can we have an additional test like:

```go
args:    []string{"zones=Availability zone 1,Availability zone 2,az2"},
```

Reply:
Absolutely! It's on the way.